server/db/models/Tab.js

```diff
@@ -0,0 +1,55 @@
+const { stateTaxTable } = require('./stateTax');
+const db = require('../db');
+const {
+  Model,
+  DataTypes: { ENUM, FLOAT },
+} = require('sequelize');
+const {
+  models: { Order },
+} = db;
+
+class Tab extends Model {
+  getTax() {
+    const venue = Venue.findAll({ where: { venueId: id } });
+    const stateVal = venue.state;
+    return (this.tax = stateTaxTable[stateVal]);
+  }
+  getSubTotal() {
+    const drinks = Order.findAll({ where: { userId: id } });
```

Comment:
'userId' should be 'id'

Reply:
sorry...wouldnt id give me the Order id? I'm looking for all of the orders where the order.userId matches the Tab.userId.